.github/workflows/gh-actions.yml

```diff
@@ -0,0 +1,31 @@
+name: Build & Test Lean
+
+on:
+  push:
+    branches: ['*']
+  pull_request:
+    branches: [master]
+
+jobs:
+  build:
+    runs-on: ubuntu-16.04
```

Comment:
Wonder why `ubuntu-16.04` seems like an old image?

Reply:
That's what is being used on Travis CI (Since, I was having issues with latest Mono runtime). I had choose this one to try out.